cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,7 +1,13 @@
 /// <reference types="Cypress" />
 
+import { MenuContentPage, CheckoutPage, PLPPage } from "../page/index";
```

Comment:
There should be more pages

Reply:
Pages added

Reply:
Do I have to add the step to the name?

Reply:
I don't understand the question, can you clarify to me please 😄 ?